Makefile

```diff
@@ -0,0 +1,16 @@
+install_dev:
```

Comment:
Personally not a huge fan of a makefile that just wraps stuff like this as it just provides unncessary abstraction over what is already a set of simple commands.